### ISSUE TITLE
Autocomplete does not find methods with multiple return values

### ISSUE THREAD

**Issue by Skysplit** [Gert-dev/php-integrator-autocomplete-plus#64]
Autocomplete is not showing methods when `@return` phpDoc block contains few pipe-separated types

eg.
```
/*
 * @return\Symfony\Component\Translation\TranslatorInterface|string
 */
```

Tested on Laravel 5.2 helpers

**Comment by Gert-dev** [comment]
This is supported, but unfortunately the docblock does not seem to be valid: there needs to be a space between the `@return` tag and the first slash and the first line needs to contain two asterisks instead of one. If there is only one asterisk, a comment is not seen as a docblock:

```php
/**
 * @return \Symfony\Component\Translation\TranslatorInterface|string
 */
```

**Comment by Skysplit** [comment]
Of course, it is like that. It was my fault, I made 2 typos in example. Sorry about that.

However, this one does not suggests methods (copied from laravel 5.2 `src/Illuminate/Foundation/helpers.php`)
```
    /**
     * Get an instance of the redirector.
     *
     * @param  string|null  $to
     * @param  int     $status
     * @param  array   $headers
     * @param  bool    $secure
     * @return \Illuminate\Routing\Redirector|\Illuminate\Http\RedirectResponse
     */
```

**Comment by Gert-dev** [comment]
That is indeed strange, autocompletion should list all possible members from all possible return types. Do both return types also exist in your codebase and are they indexed (e.g. can you alt-click them to navigate to them)?

Does it also fail with some built-in types such as `@return * @return \DateTime|\SplFileObject`?

**Comment by Skysplit** [comment]
`alt-click` works fine. It navigates me to class' source file.

Yes, it also fails for built-in types.

Example:

```php
/**
 * @return \SplFileObject|\DateTime
 */
function testFunction()
{

}

/**
 * @return \SplFileObject
 */
function anotherTestFunction()
{

}

testFunction()-> // no suggestions
anotherTestFunction-> // suggestions appear properly

**Comment by Gert-dev** [comment]
I just remembered, whilst reading through the development changelog, that you are absolutely correct: global functions with multiple return types weren't being properly analyzed. I've already fixed this on the development branch (and I was using it for testing), which is why I couldn't reproduce. This will be fixed in the upcoming 1.2.0 release.

**Comment by Skysplit** [comment]
Thank you. Cannot wait for next release then :) Your package imho is absolute must for any php developer that uses atom

**Comment by Gert-dev** [comment]
The fix is included in the just-released 1.2.0 version of the base package. Let me know if you have any more problems.

**Comment by Skysplit** [comment]
Looks like it's working properly. Thank you very much!
